### ISSUE TITLE
LiveTV Transcoding kills HLS "Fragment-Reload"

### ISSUE THREAD

**Issue by morbificagent** [jellyfin/jellyfin#4511]
Hi Together,

i think there is a problem with the transcoding of LiveTV HLS Streams.
The transcoder seems to grab the hls fragments which are accessable at the moment of transcoding-start and i get a transcoded video.
But it doesnt grab the next fragments so the livetv stops after a few seconds. :-(

The only way at the moment seems to disable transcoding... but as this isnt selectable for livetv only i have to disable it for the whole userprofile.  :-( too

Am i doing anything wrong and/or is there a better workaround anyone know?

**Comment by Artiume** [comment]
Within your M3U Tuner, have you tried enabling the option

Auto-loop live streams
Enable this if live streams only contain a few seconds of data and need to be continuously requested. Enabling this when not needed may cause problems.

**Comment by morbificagent** [comment]
Yes i have... Because of my m3u is configured to list only 3 fragments of the stream it wouldn't work without that.

**Comment by Artiume** [comment]
Perhaps it needs to be tweaked to get the right performance for your particular setup. @crobibero you have any ideas?

**Comment by crobibero** [comment]
Is this unstable or 10.6.4? I haven’t had any issues with my LiveTV setup on either (after fixing master)

**Comment by morbificagent** [comment]
Hi, its 10.6.4

**Comment by morbificagent** [comment]
Jeeeha, the hint with build the git-version resolves the problem! :-)
Many thanks and great work!

Have a nice week

**Comment by morbificagent** [comment]
Sorry have to ask a question again...
I have compiled the git version and livetv was working great. But there were so many problems with it as Kodi couldnt connect anymore like the AndroidTV-App too... And all of my plugins where gone and i could not install them again.
And the Network-Settings i made in the webinterface werent written to the right config-file.
All in all everything was a mess after that.

I ended in rolling back to the 10.6.4.

Did i anything wrong in compiling the git-version of 10.7 and installing the .deb over my existing 10.6.4? Or is it too much "beta"

And if it is like that, would it be possible to copy only the files which are for the live-tv part from the compiled .deb to the 10.6.4 so i could keep the working version and only change its live-tv part?
